server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/rule/OracleAffectedRowsExceedLimit.java

```diff
@@ -115,7 +115,7 @@ private long getOracleAffectedRows(String originalSql, JdbcOperations jdbcOperat
                 "SELECT * FROM TABLE(DBMS_XPLAN.DISPLAY('PLAN_TABLE', '" + ODC_TEMP_EXPLAIN_STATEMENT_ID + "', 'ALL'))";
         List<String> queryResults = jdbcOperations.query(getPlanSql, (rs, rowNum) -> rs.getString("PLAN_TABLE_OUTPUT"));
         long estRowsValue = 0;
-        for (int rowNum = 5; rowNum < queryResults.size(); rowNum++) {
+        for (int rowNum = 2; rowNum < queryResults.size(); rowNum++) {
```

Comment:
why 2?

Reply:
![image](https://github.com/user-attachments/assets/988dad86-2453-482c-b4c8-8fe9a61ba508)
4 or before 4 is fine

Reply:
why do you set 5 before? does ob-mysql has the same problems?

Reply:
no

Reply:
![image](https://github.com/user-attachments/assets/fb44e4fc-95ec-4a1b-b963-6990a4689be6)
The index will not get the value of column row from index 5

Reply:
from row 0 may be better?